openapi.json

```
@@ -1229,6 +1229,51 @@
         }
       }
     },
+    "/accounts/{account_id}/line_items/refunds": {
+      "post": {
+        "tags": ["line_items"],
+        "summary": "Create a refund for a specific account",
+        "description": "",
+        "operationId": "createAccountRefund",
+        "parameters": [
+          {
+            "name": "account_id",
+            "in": "path",
+            "description": "Account ID",
+            "required": true,
+            "style": "simple",
+            "explode": false,
+            "schema": {
+              "type": "string",
+              "example": "31mNprzLd2bKl6koVna68ARM"
```

we should use a UUID as the example now 
```suggestion
              "example": "f3991696-ad13-41bf-864d-178a8079409b"
```